adapters/ix/ix.go

```diff
@@ -309,3 +326,34 @@ func marshalJsonWithoutUnicode(v interface{}) (string, error) {
 	// https://pkg.go.dev/encoding/json#Encoder.Encode
 	return strings.TrimSuffix(sb.String(), "\n"), nil
 }
+
+func BuildIxDiag(request *openrtb2.BidRequest) error {
+	extRequest := &ExtRequest{}
+	if request.Ext != nil {
+		if err := json.Unmarshal(request.Ext, &extRequest); err != nil {
+			return err
+		}
+	}
+	ixdiag := &IxDiag{}
+
+	if extRequest.Prebid != nil {
+		ixdiag = &IxDiag{
+			PbjsV: extRequest.Prebid.Channel.Version,
+		}
+	}
+
+	// Slice commit hash out of version
+	if strings.Contains(version.Ver, "-") {
+		ixdiag.PbsV = version.Ver[:strings.LastIndex(version.Ver, "-")]
+	} else if version.Ver != "" {
+		ixdiag.PbsV = version.Ver
```

Comment:
`version.Ver` gets populated in runtime with the return value of `git describe --tags` that may return a string with two hyphens:
```
╰─➤  git describe --tags
0.23.1-3-g4ee257d8
```
Can we please modify this logic to correctly parse a string like this?